Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.cs

```diff
@@ -338,6 +341,33 @@ public string ExportAsJson()
             return _drawingSurfaceRenderer.GetSerializedList();
         }
 
+        /// <summary>
+        /// Export the InfinitCanvas as PNG file.
+        /// </summary>
+        /// <returns>json string</returns>
+        public Task ExportAsPng(IStorageFile file)
+        {
+            return _drawingSurfaceRenderer.ExportAsPNG(file);
+        }
+
+        /// <summary>
+        /// Export the InfinitCanvas ink strokes.
+        /// </summary>
+        /// <returns>json string</returns>
+        public List<InkStroke> ExportInkStrokes()
```

Comment:
Return is list of InkStroke not json string

Reply:
@IbraheemOsama can you clarify, this appears to already export `InkStroke`?  Do you mean it should instead be returning serialized json string like `ExportAsJson` but filtered to strokes?  Similar with the one below?